crates/wasmtime/src/runtime/store.rs

```diff
@@ -76,22 +76,22 @@
 //! contents of `StoreOpaque`. This is an invariant that we, as the authors of
 //! `wasmtime`, must uphold for the public interface to be safe.
 
-use crate::hash_set::HashSet;
 use crate::instance::InstanceData;
 use crate::linker::Definition;
 use crate::module::RegisteredModuleId;
 use crate::prelude::*;
 use crate::runtime::vm::mpk::ProtectionKey;
+#[cfg(feature = "gc")]
+use crate::runtime::vm::GcRootsList;
```

Comment:
I thought you wanted to fully qualify these at the use site?

Reply:
I guess there are multiple uses in this module or something like that?

Reply:
Ideally yeah I generally prefer avoiding `#[cfg]` on imports since it feels pretty noisy, but this one was used so much it was getting even noisier to have fully qualified use-sites.

Probably an indicator to create `crates/wasmtime/src/runtime/store/gc.rs` perhaps...